src/ffi/cpython/mod.rs

```diff
@@ -0,0 +1,3 @@
+pub mod abstract_;
```

Comment:
:+1:  for creating this new module. Maybe we can reduce `cfg` using this.

Reply:
Yes, I think for sure (again see #1289).